api/src/Command/UserRetentionPolicyCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Entity\User;
+use App\Event\UserRetentionPolicyCommandEvent;
+use App\EventDispatcher\ObservableEventDispatcher;
+use App\Repository\UserRepository;
+use App\Service\Audit\AuditEvents;
+use App\v2\Controller\ControllerTrait;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UserRetentionPolicyCommand extends Command
+{
+    use ControllerTrait;
+
+    public static $defaultName = 'digideps:user-retention-policy';
+
+    public function __construct(
+        private UserRepository $userRepository,
+        private ObservableEventDispatcher $eventDispatcher,
+        private LoggerInterface $logger
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Deletes inactive admin user accounts')
+        ;
+    }
+
+      protected function execute(InputInterface $input, OutputInterface $output)
+      {
+          /** @var User $user */
+          $getInactiveAdminUsers = $this->userRepository->getAllAdminAccountsNotUsedWithin('-24 months');
+
+          $inactiveAdminUserIds = [];
+
+          if ($getInactiveAdminUsers) {
+              if (is_array($getInactiveAdminUsers)) {
+                  foreach ($getInactiveAdminUsers as $adminUser) {
+                      $inactiveAdminUserIds[] = $adminUser->getId();
+                      $this->auditLogDeletionAutomation($adminUser);
+                  }
+              } elseif ($getInactiveAdminUsers instanceof User) {
+                  $inactiveAdminUserIds[] = $getInactiveAdminUsers->getId();
+                  $this->auditLogDeletionAutomation($getInactiveAdminUsers);
+              }
+          }
+
+          if (!empty($inactiveAdminUserIds)) {
+              $countOfAdminUsers = count($inactiveAdminUserIds);
+
+              $this->userRepository->deleteInactiveAdminUsers($inactiveAdminUserIds);
+              $output->writeln(sprintf('%d inactive admin user(s) deleted', $countOfAdminUsers));
```

Comment:
No longer returning true if the command successfully runs?

Reply:
If I remove the return statement it will delete the inactive admin user, however it also throws an error as the command is expecting a return value.